frontend/src/bundles/cv-editor/components/user-flow-bar/user-flow-bar.tsx

```diff
@@ -0,0 +1,56 @@
+import { useState } from 'react';
+
+import { UserFlowBtns } from './types/interface';
+import css from './user-flow-bar.module.css';
+
+const UserFlow: React.FC = () => {
+    const [isActiveStep, setisActiveStep] = useState('personal');
+
+    const buttonArray = Object.keys(UserFlowBtns);
+
+    const onTabClick = (step: string) => (event_: React.MouseEvent) => {
+        event_.preventDefault();
+        setisActiveStep(step);
+    };
+
+    const onNextClick = (array: string[]) => (event_: React.MouseEvent) =>  {
+        event_.preventDefault();
+        const index = array.indexOf(isActiveStep);
+        setisActiveStep(array[index+1]);
+        if (index === array.length-1) {
+            setisActiveStep('personal');
+        }
+        //add logic to save fullfilled data and to connect it with progress bar
+    };
+
+    return (    
+        <section className={css.userFlow}>
+            <aside className={css.userFlow__sidebar}>
```

Comment:
Section and Aside are interchangeable tags, and none of them should be used in this component